portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,157 @@
-#content {
+@import url('https://fonts.googleapis.com/css2?family=Titillium+Web:wght@600&display=swap');
+.main
+{
+    align-items: center;
+}
+
+.navbar
+{
+    border-bottom-left-radius:10px;
+    border-bottom-right-radius: 10px; 
+    margin-left: 5%;
+    margin-right: 5%;
+    color: white;
+    background: #193341;
+    box-sizing: border-box;
+    padding-top: 1px;
+    align-items: center;
+    justify-content: center;
+}
+
+.nav_elements
+{
+    align-self: center;
+    color: white;
+}
+
+a
+{
+    text-decoration: none;
+    color: inherit;
+}
+
+.nav_elements:hover
+{
+    color: white;
+    text-emphasis-color: white;
+}
+
+.line
+{
+    align-self: right;
+    color: rgb(238, 61, 61);
+    width:70%;
+    height: 2%;
+}
+.dot
+{
+  height: 25px;
+  width: 25px;
+  border-radius: 50%;
+  display: inline-block;
+}
+
+.dot1
+{
+    background-color: white;
+}
+
+.dot2
+{
+    background-color: #204051;
+}
+
+.navbar:hover
+{
+    opacity: 0.7;
+}
+
+.navbar
+{
+    display: flex;
+    color: white;
+}
+
+.container
+{
+    padding: 5% 5% 5% 5%;
+    display: block;
+    align-items: center;
+    justify-content: center;
+    margin-left: 20%;
+    margin-right: 20%;
+    text-align: center;
+}
+
+.container1
+{
+    color: white;
+    background: #204051;
+}
+
+.container2
+{
+    color: #204051;
+    background: white;
+}
+
+.img
+{
+    border-radius: 50%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
+
+.fa {
+    display: flex;
+  font-size: 70%;
+  padding: 2% 2% 2% 2%;
+  width:3%;
+  height:5%;
+  text-align: center;
+  text-decoration: none;
   margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-right: auto;
+  border-radius: 50%;
+  align-items: center;
+}
+
+.fa:hover {
+    opacity: 0.7;
+}
+
+.fa-facebook {
+  background: #3B5998;
+  color: white;
+}
+
+.fa-twitter {
+  background: #55ACEE;
+  color: white;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.fa-google {
+  background: #dd4b39;
+  color: white;
 }
+
+.fa-linkedin {
+  background: #007bb5;
+  color: white;
+}
+
+.fa-github
+{
```

Comment:
follow a consistent style. Either keep brackets in the same line as the class everywhere or the other way round.
Just use a CSS formatting tool and all these should be fine : https://www.cleancss.com/css-beautify/